app/models/spree/country.rb

```diff
@@ -6,7 +6,7 @@ class Country < ApplicationRecord
 
     validates :name, :iso_name, presence: true
 
-    def self.cached_find_by(attrs)
+    def self.find_by_cached(attrs)
```

Comment:
Your commit message says what you are doing here but not why. Generally, I can see the "what" in the diff but the "why" is the important thing to communicate to others. Is there a collision with Rails method naming? If so, should we explain what the difference between our version and Rails' version is?

Reply:
Sorry, I forgot to explain.

I noticed the collusion after reading this [article](https://blog.saeloun.com/2022/02/09/rails-prevent-duplicates-in-find_by-cache/)
I double checked on Rails repo 
https://github.com/rails/rails/blob/1ea796d6a887ddb6fd7a2395399ae35871aeb00c/activerecord/lib/active_record/core.rb#L246-L257